iotdb-core/datanode/src/main/java/org/apache/iotdb/db/schemaengine/schemaregion/mtree/impl/pbtree/CachedMTreeStore.java

```diff
@@ -556,6 +557,7 @@ public void clear() {
         }
       }
       file = null;
+      PageLifecycleManager.getInstance().removePagePool(schemaRegionId);
```

Comment:
Should this be placed in file.clear()?

Reply:
> Should this be placed in file.clear()?

Yes. But schema region id is invisible in file...